docs/application/dotnet/guides/nui/relative-layout.md

```diff
@@ -0,0 +1,84 @@
+The width of `RedRect` and `BlueRect` is set as `100` by specifying the `WidthSpecification` property. However, the width of `GreenRect` and `YellowRect` is resized within the layout space that is set by the `LeftTarget` and `RightTarget`.
+
+The following table describes the properties of `RelativeLayout`:
+
+| Property               | Type            | Description  |
+| -----------------------| --------------- | ------------ |
+| `LeftTarget`           | View            | Gets or sets the target that the child's left side is referring to.|
+| `RightTarget`          | View            | Gets or sets the target that the child's right side is referring to.|
+| `TopTarget`            | View            | Gets or sets the target that the child's top side is referring to.|
+| `BottomTarget`         | View            | Gets or sets the target that the child's bottom side is referring to.|
+| `LeftRelativeOffset`   | float           | Gets or sets the relative offset for the left target. <br /> When the value is `0`, the left edges of the left target and child view are aligned.<br/> When the value is `1`, the left edge of the child view is aligned to the right edge of the left target. |
+| `RightRelativeOffset`  | float           | Gets or sets the relative offset for the right target. <br /> When the value is `0`, the right edge of the child view is aligned to the left edge of the right target.<br/> When the value is `1`, the right edges of the right target and child view are aligned.
+| `TopRelativeOffset`    | float           | Gets or sets the relative offset for the top target. <br /> When the value is `0`, the top edges of the top target and child view are aligned.<br/> When the value is `1`, the top edge of the child view is aligned to the bottom edge of the top target.
+| `BottomRelativeOffset` | float           | Gets or sets the relative offset for the bottom target. <br /> When the value is `0`, the bottom edge of the child view is aligned to the top edge of the bottom target.<br/>When the value is `1`, the bottom edges of the bottom target and child view are aligned.
+| `HorizontalAlignment`  | RelativeLayout.Alignment| Gets or sets the horizontal alignment of the child view.|
+| `VerticalAlignment`  | RelativeLayout.Alignment| Gets or sets the vertical alignment of the child view.|
+| `FillHorizontal`  | bool | Gets or sets the boolean value. It indicates whether or not the child fills its horizontal space.|
+| `FillVertical`  | bool | Gets or sets the boolean value. It indicates whether or not the child fills its vertical space.|
+
+## Layout space of child view
+
+The `Target` and the `RelativeOffset` properties determine the size of the layout space. The `Target` property indicates the target view in which each edge of the layout space is aligned. The default value is `null`, which refers to the parent's relative layout.
```

Comment:
Preposition at the end of the sentence: Matter of choice. 
Example
Where is it?
Where is it at?

`Target` indicates the target view that each edge of layout space is aligned with. the default is `null` that means parent relative layout.


I added **value** too, 
Just to make comprehension much easier and better the sentence structure, However, if you want, we can go with your suggestion. 


We would prefer ist one?

Reply:
@nawab-samsung if it was "to which" it would you have been accurate not "in which" 

so my suggestion would be either one:

The `Target `property indicates the target view to which each edge of the layout space is aligned.  

or 

Target indicates the target view that each edge of layout space is aligned with. the default is null that means parent relative layout.

Reply:
I had suggested the following: 

![image](https://user-images.githubusercontent.com/43839737/107607157-71fc0700-6c5e-11eb-8542-c329274265b3.png)


Then after the push, developer had made it to:

The `Target` indicates the target view in which each edge of the layout space is aligned. The default value is `null`, which refers to the parent's relative layout.


I didn't find it grammatically wrong, so didn't change.